docs/docs/getting-started/02-oauth-tutorial.mdx

```diff
@@ -253,11 +253,13 @@ Once inserted and correct, Github will redirect the user to our app and Auth.js
 <img src={nextAuthUserLoggedIn} />
 
 Great! We have completed the whole E2E authentication flow setup so that users can login in our application through Github!
+:::
 
 :::info
-You can create your own Sign In page instead of using the default one from Auth.js. You can learn how to do so in our dedicated guide for it.
+You can create your own Sign In page instead of using the default one from Auth.js. You can learn how to do so in our [dedicated guide for it](https://authjs.dev/guides/basics/pages).
```

Comment:
```suggestion
You can create your own Sign In page instead of using the default one from Auth.js. You can learn how to do so in our [dedicated guide for it](/guides/basics/pages).
```